src/site/content/en/metrics/user-centric-metrics/index.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: User-centric metrics
+authors:
+  - philipwalton
+date: 2019-11-07
+description: |
+  User-centric metrics are a critical tool in understanding and improving the
+  experience of your site in a way that matters and benefits real users.
+---
+
+We've all heard how important performance is. But when we talk about
+performance—and about making websites "fast"—what specifically do we mean?
+
+The truth is performance is relative:
+
+- A site might be fast for one user (on a fast network with a powerful device)
+  but slow for another user (on a slow network with a low-end device).
+- Two sites may finish loading in the exact same amount of time, yet one may
+  _seem_ to load faster (if it loads content progressively rather than waiting
+  until the end to display anything).
+- A site might _appear_ to load quickly but then respond slowly (or not at all)
+  to user interaction.
+
+So when talking about performance, it's important to be precise and to refer to
+performance in terms of objective criteria that can be quantitatively measured.
+These criteria are known as _metrics_.
+
+But just because a metric is based on objective criteria and can be
+quantitatively measured, it doesn't necessarily mean those measurements are
+useful.
```

Comment:
Are you still happy with this intro, given what you said in our chat?

> BTW, I'm realizing that I too often describe metrics as a means to measure performance, so I'm trying to go through and reframe certain parts (specifically in the custom metrics doc) to frame them as a tool to measure the user experience.

This intro heavily frames metrics as something related to performance